src/content/self-hosted/aks/config.yaml

```diff
@@ -14,13 +14,14 @@ auth:
 cluster:
   endpoint:
 
-cloud:
-  provider:
-    azure:
-      enabled: true
-      storage:
-        container: ""
-        accountName: ""
+registry:
```

Comment:
@rberrelleza @huguestennier looks like this config file, that the installation guide states to download, it's shared between all the okteto doc versions.
Could we have a dedicated config file for each version? Otherwise, updating this file can break the previous installation guides.

Reply:
Yeah, that's a good idea. We should be able to do it. I'll do a separate PR.

Reply:
@pchico83 what are the changes needed for the 1.7 docs?

Reply:
@rberrelleza nothing blocking. But I think it's a good improvement is the config file URL are also versioned